django/db/models/indexes.py

```diff
@@ -0,0 +1,57 @@
+
+__all__ = ['Index']
+
+
+class Index(object):
+
+    def __init__(self, *fields, **kwargs):
+        self.fields = fields
+        if 'model' in kwargs:
+            self.model = kwargs['model']
+        if 'name' in kwargs:
+            self.name = kwargs['name']
+
+    def as_sql(self, schema, suffix=''):
```

Comment:
Thinking about it, I'm not sure `as_sql` is the right name, given `drop_sql`. Perhaps `create_sql` would be better?